lib/services/cordova-project-plugins-service.ts

```diff
@@ -0,0 +1,682 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import {EOL} from "os";
+import * as util from "util";
+import * as pluginsDataLib from "../plugins-data";
```

Comment:
As you are only using `PluginType` from `plugins-data` it can be captured as follows:
```Typescript
import {PluginType} from "../plugins-data";
```